packages/react-notion-custom/src/lib/components/column-list.tsx

```diff
@@ -0,0 +1,19 @@
+import { ColumnListArgs } from "../types";
+
+const ColumnList: React.FC<{ children: React.ReactNode } & ColumnListArgs> = ({
+  children,
+  ...props
+}) => {
+  const columnCount = props.blocks?.length || 0;
```

Comment:
p0: `||` replaces all falsy values (0, '', false, null, undefined), but `??` only replaces null or undefined. While there’s no functional issue, conceptually, it seems better to use `??`.

Reply:
I fixed it. thank you